README.md

```diff
@@ -1,5 +1,40 @@
-# FCC Test Suite Template
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+# Project Name
 
-You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+> "Review-past-code" is the repository for my portfalio 
+
+## Built With
+
+- HTML y CSS
+- Git, Github and Visual Studio Code
+-Review past code 
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Choose a directory
+- Open a Terminal
+- write: git clone https://github.com/M1rn4/Review-past-code
+- get into to directory "Review-past-code"
+- write: npm install
+
+## Author
+
```

Comment:
This is an example of inline comment . check in provided guideline, you will see how to create inline comment.